src/test/java/seedu/address/testutil/TypicalIndexes.java

```diff
@@ -13,4 +13,5 @@
     public static final Index NOTE_SECOND_INDEX = Index.fromOneBased(2);
     public static final Index TASK_FIRST_INDEX = Index.fromOneBased(1);
     public static final Index TASK_SECOND_INDEX = Index.fromOneBased(2);
+    public static final Index TASK_THIRD_INDEX = Index.fromOneBased(2);
```

Comment:
Apologies, what's the point of this variable if it is the same as TASK_SECOND_INDEX?

Reply:
Thanks for the catch! I was trying out different variables to implement some tests and I think I missed this